src/app/views/home/home-view.tsx

```diff
@@ -18,21 +18,7 @@ export interface TabInfo {
   downloadLink: string;
 }
 
-interface TabItemProps {
```

Comment:
I think having these as interfaces is much more readable than putting them as types of the arguments in the component